torch/csrc/jit/fuser/compiler.cpp

```diff
@@ -144,6 +148,85 @@ static void setInputBroadcastGroups(KernelSpec& spec) {
       std::back_inserter(spec.inputBroadcastGroups()));
 }
 
+// This function moves GradSumToSize nodes inside a fusion group to the end of
+// the fusion group. Note that correctness relies on the invariant that
```

Comment:
> to the end of the fusion group

Are they being moved outside the fusion group?

Reply:
They're eliminated by pushing them to the outputs of the fusion group, recording their effect on the outputs, and deleting them. I'll update the comment.